hoomd/hpmc/tune/move_size.py

```diff
@@ -136,14 +84,12 @@ def __init__(self,
         # This is a bit complicated because we are having to ensure that we keep
         # the list of tunables and the solver updated with the changes to
         # attributes. However, these are simply forwarding a change along.
-        param_dict = ParameterDict(
-            moves=OnlyIf(to_type_converter([OnlyFrom(['a', 'd'])]),
-                         postprocess=self._update_moves),
-            types=OnlyIf(to_type_converter([str]),
-                         postprocess=self._update_types,
-                         allow_none=True),
-            target=OnlyTypes(float, postprocess=self._target_postprocess),
-            solver=SolverStep)
+        param_dict = ParameterDict(moves=OnlyIf(to_type_converter(
+            [OnlyFrom(['a', 'd'])]),
+                                                postprocess=self._update_moves),
+                                   types=OnlyIf(to_type_converter([str]),
+                                                postprocess=self._update_types,
+                                                allow_none=True))
```

Comment:
```suggestion
                                                allow_none=True)),
```
Maybe yapf will format this cleaner with the trailing comma?